services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_scheduler_dask.py

```diff
@@ -129,6 +129,7 @@ async def _start_tasks(
                         hardware_info=task.hardware_info,
                         callback=wake_up_callback,
                         metadata=comp_run.metadata,
+                        run_id=comp_run.run_id,
```

Comment:
so this is not the correct run_id for RUT.
https://github.com/ITISFoundation/osparc-simcore/blob/fd62ccf073ed9d0faa98ea89707cb823e7dee51d/services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/_utils.py